hydra_python_core/doc_writer.py

```diff
@@ -519,13 +518,13 @@ def generate(self) -> Dict[str, Any]:
 class HydraStatus():
     """Class for possibleStatus in Hydra Doc."""
 
-    def __init__(self, code: str, title: str, desc: str) -> None:
+    def __init__(self, code: int, title: str = "", desc: str = "") -> None:
         """Create method."""
         self.code = code
         self.title = title
         self.desc = desc
 
-    def generate(self) -> Dict[str, Any]:
+    def generate(self, status_type: str = "Status") -> Dict[str, Any]:
```

Comment:
We need to add a field for identifiers, if it is needed. This had to be done using the `@id` field.

Reply:
The `@id` field should be optional and only returned in the generated dictionary if assigned.